api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -763,4 +726,15 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
 
     return ResponseEntity.ok(fileDetail);
   }
+
+  @Override
+  public ResponseEntity<WorkspaceUserRolesResponse> getWorkspaceUserRoles(
+      String workspaceNamespace, String workspaceId) {
+    org.pmiops.workbench.db.model.Workspace dbWorkspace =
+        workspaceService.getRequired(workspaceNamespace, workspaceId);
+    List<UserRole> userRoles = workspaceService.getWorkspaceUserRoles(dbWorkspace);
```

Comment:
I would pass in the map here, and have this just be a helper that converts a Map<User, Acl> to List<Acl>